src/Sylius/Behat/Context/Setup/ShippingContext.php

```diff
@@ -124,6 +124,25 @@ public function shippingMethodBelongsToTaxCategory(ShippingMethodInterface $ship
         $this->shippingMethodManager->flush();
     }
 
+
+    /**
+     * @Transform /^"([^"]+)" shipping to "([^"]+)"$/
```

Comment:
Instead of getting shipping method by zone we should get it simply by name and add a shipping and billing address to the order with country specified after `shipping to "MyCountry"`.